core/reflection.jvm/resources/META-INF/proguard/kotlinreflect.pro

```diff
@@ -0,0 +1,14 @@
+##---------------Begin: Kotlin-reflect ----------
+# Keep Metadata annotations so they can be parsed at runtime.
+-keep class kotlin.Metadata { *; }
+
+# Keep kotlin-reflect internals.
+-keep class kotlin.reflect.jvm.** { *; }
+
+# Keep generic signatures and annotations at runtime.
+# R8 requires InnerClasses and EnclosingMethod if you keepattributes Signature.
+-keepattributes InnerClasses,Signature,*Annotation*,EnclosingMethod
```

Comment:
Do we need broad *Annotation* or just RuntimeVisibleAnnotation (because metadata is runtime visible) ?

Reply:
I didn't know that `RuntimeVisibleAnnotations` was a thing! Historically any project reading annotations at runtime has always used `*Annotation*`, but maybe that's just a bespoke pattern that isn't needed. Let me try in our codebase

Reply:
There are also `RuntimeVisibleParameterAnnotations` and `RuntimeVisibleTypeAnnotations`, so maybe the pattern should be `RuntimeVisible*Annotations`, if that works...

Reply:
That seems to work b843558